makefiles/boot/riotboot.mk

```diff
@@ -74,9 +74,11 @@ $(BINDIR_APP)-slot1.hdr: OFFSET=$(SLOT1_IMAGE_OFFSET)
 # Generic target to create a binary files for both slots
 riotboot: $(SLOT_RIOT_BINS)
 
+# Same as CLEAN for the submake riotboot/bootlader/% targets
+RIOTBOOT_CLEAN = $(if $(filter clean, $(MAKECMDGOALS)),riotboot/bootloader/clean,)
```

Comment:
Isn't there a CLEAN variable that can be used `(if not empty $CLEAN, riotboot/bootloader/clean)`?